_posts/hi/newsletters/2022-07-20-newsletter.md

```diff
@@ -0,0 +1,165 @@
+
+            Maxwell ने आगे नोट किया कि Bitcoin खनिक औसतन केवल लेनदेन शुल्क के माध्यम से काफी
+            अधिक मूल्य एकत्र करते हैं, जो कि कई altcoins सब्सिडी, शुल्क और अन्य प्रोत्साहनों के
+            संयोजन के माध्यम से अपने खनिकों को भुगतान करते हैं। उन altcoins को मौलिक PoW
+            हमलों का सामना नहीं करना पड़ रहा है, जिसका अर्थ है कि यह हो सकता है कि Bitcoin
+            को सुरक्षित रखने के लिए अकेले लेनदेन शुल्क के माध्यम से पर्याप्त मूल्य का
+            भुगतान किया जा रहा हो। संक्षेप में, Bitcoin पहले से ही उस बिंदु से आगे निकल सकता
+            है जहां उसे पर्याप्त PoW सुरक्षा प्राप्त करने के लिए अपनी सब्सिडी की आवश्यकता
+            होती है। (हालांकि सब्सिडी वर्तमान में अन्य लाभ भी प्रदान करती है, जैसा कि Bram Cohen के
+            thread के लिए नीचे सारांश में चर्चा की गई है।)
+
+            Towns बताते हैं कि Peter Todd के परिणाम प्रत्येक वर्ष खो जाने वाले Bitcoin की एक निरंतर
+            औसत दर पर निर्भर करते हैं, लेकिन Towns को लगता है कि यह खोए हुए Bitcoin
+            को कम करने के लिए एक सिस्टम-वाइड लक्ष्य होना चाहिए। संबंधित रूप से, Maxwell
+            वर्णन करता है कि किसी को भी स्वचालित रूप से एक स्क्रिप्ट का उपयोग करने की
+            अनुमति देकर सिक्के के नुकसान को लगभग पूरी तरह से समाप्त किया जा
+            सकता है, जो उनके किसी भी सिक्के को दान करेगा, जो कि 120 वर्षों के लिए स्थानांतरित
+            नहीं हुआ था --- अच्छी तरह से अपेक्षित जीवनकाल से पहले मूल स्वामी और
+            उनके उत्तराधिकारियों की।
+
+        - *<!--censorship-resistance-->सेंसरशिप प्रतिरोध:* डेवलपर ZmnSCPxj [विस्तारित][zmnscpxj cr]
+          एक [तर्क][voskuil cr] Eric Voskuil द्वारा कि लेनदेन शुल्क Bitcoin
+          के सेंसरशिप प्रतिरोध को बढ़ाता है। उदाहरण के लिए, यदि ब्लॉक इनाम का 90% सब्सिडी से
+          आता है और 10% लेनदेन शुल्क से आता है, तो एक सेंसरिंग माइनर को सबसे अधिक राजस्व
+          सीधे 10% का नुकसान हो सकता है। लेकिन अगर 90% फीस से और 10% सब्सिडी से आता है, तो खनिक
+          सीधे 90% तक खो सकता है --- सेंसरशिप से बचने के लिए एक बहुत मजबूत प्रोत्साहन है।
+
+            Peter Todd [काउंटर][todd cr] इस राय के साथ कि एक सतत जारी करने से PoW सुरक्षा के
+            लिए "piddling transaction fees" की तुलना में अधिक धन जुटाया जाएगा, और यह कि
+            एक उच्च ब्लॉक इनाम लागत में वृद्धि करेगा जो एक हमलावर को सेंसर लेनदेन के लिए खनिकों को
+            भुगतान करना होगा। .
+
+    - [<!--fee-sniping-->फीस स्निपिंग][cohen fs]: Bram Cohen ने [फीस स्निपिंग][topic fee sniping]
+      की समस्या के बारे में पोस्ट किया और एक संभावित समाधान के रूप में कुल ब्लॉक पुरस्कारों
+      (शेष सब्सिडी) के लगभग 10% पर लेनदेन शुल्क रखने का सुझाव दिया। उन्होंने
+      कुछ अन्य संभावित समाधानों का संक्षेप में उल्लेख किया, लेकिन अन्य ने अधिक विस्तार से
+      अतिरिक्त सुझाव दिए।
+
+        - *<!--paying-fees-forward-->आगे शुल्क का भुगतान करना:* Russell O'Connor [आगे रखना][oconnor forward fees]
+          एक पुराना विचार है कि खनिक फीस काटने को प्रोत्साहित किए बिना अपने mempool में शीर्ष लेनदेन से अधिकतम शुल्क की 
```

Comment:
Our markdown formatting checks for trailing spaces and there was an extra trailing space here. Ive pushed a small fixup to remove it to fix the build.

Reply:
Thanks, @bitschmidty!